javascript/javascript3/week1/weather-app.js

```diff
@@ -0,0 +1,65 @@
+const weatherInput = document.getElementById("weather-input");
+const weatherButton = document.getElementById("input-button");
+
+const loading = document.getElementById("loading");
+loading.style.visibility = "hidden"; //loading icon is hidden
+loading.style.width = "5%";
+
+const myKey = "3cebafb43fa594f9f6c39fac18a52ac7";
+
+weatherButton.addEventListener("click", showWeather);
+
+function showWeather() {
+  loading.style.visibility = "visible"; // loading icon is visible
```

Comment:
Another CSS property is `display: none` and `display: inline` that you can play around with that is slightly different from visibility.